components/RequestModal/index.js

```diff
@@ -41,6 +43,8 @@ const RequestModal = ({ mentor, isOpen, onClose }) => {
         longterm: data.longterm,
       }),
     });
+
```

Comment:
One empty line here instead of two?

Reply:
Ah we should setup some linting/prettier rules on save/commit